config/deploy.rb

```diff
@@ -1,24 +1,42 @@
 set :application, 'morph'
-set :repo_url, 'https://github.com/openaustralia/morph.git'
+set :repo_url, `git config --get remote.origin.url`.strip
```

Comment:
ah, this means it uses whatever you have configured locally. Not sure how this is better.